code/game/mecha/mecha.dm

```diff
@@ -639,6 +639,7 @@
 			AI.control_disabled = 1
 			AI.radio_enabled = 0
 			AI.loc = card

```

Comment:
no it is an issue, you just need to *replace* the code that does AI.loc = card
with the forceMove

Same goes with the rest of the instances